src/node/NodeOptions.hs

```diff
@@ -222,6 +207,25 @@ addrNodeOption =
         metavar "HOST:PORT" <>
         help "Identifier of a node in a Kademlia network"
 
+tlsParamsOption :: Parser TlsParams
```

Comment:
@Martoon-00 aha, light wallet doesn't have any implementation as of yet (at least from the code you have pasted). That being said, we can then ignore TLS there